cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -383,10 +383,10 @@ ReconciliationState createReconciliationState(Reconciliation reconciliation, Kaf
         /* test */ Set<String> fsResizingRestartRequest = new HashSet<>();
 
         // Certificate change indicators
-        private boolean isZookeeperCertsChanged = false;
-        private boolean isKafkaCertsChanged = false;
-        private boolean isKafkaExporterCertsChanged = false;
-        private boolean isEntityOperatorCertsChanged = false;
+        private boolean isExistingZookeeperCertsChanged = false;
+        private boolean isExistingKafkaCertsChanged = false;
+        private boolean isExistingKafkaExporterCertsChanged = false;
+        private boolean isExistingEntityOperatorCertsChanged = false;
```

Comment:
If we're changing the name of these anyway, I wonder if we should drop the `is` part. The tense disagrees with the `changed` suffix, so it reads a bit awkwardly.

Reply:
Yeah, now when you say it it sounds weird :-D